app/views/posts/_post.html.erb

```diff
@@ -38,8 +38,8 @@
     <% end %>
 
     <% if post.comments.any? && !hide_feedbacks %>
-      <span class="comment-hint text-muted" title="<%= pluralize(post.comments.count, 'comment') %>">
-        <span class="glyphicon glyphicon-comment"></span> &times;<%= post.comments.count %>
+      <span class="comment-hint text-muted" title="<%= pluralize(post.comments.size, 'comment') %>">
```

Comment:
This won't remove the additional query here. Rails overrides `size` to be the same as `count`, so it still does the query. You want to add `.includes(:comments)` to the query in the controller (or add `:comments` to an existing `includes`) , then use the normal `count` here.